src/logic/calculate.js

```diff
@@ -0,0 +1,96 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
+export default function calculate(obj, buttonName) {
+  if (buttonName === 'AC') {
+    return {
+      total: null,
+      next: null,
+      operation: null,
+    };
+  }
```

Comment:
- [ ] Consider displaying a default zero when you open an application and after performing an operation when you click `AC` you can display a zero

Reply:
Dear @DeliceLydia,

I have fixed your notes. Thanks for review 🥇